forward/ft_headmodel_openmeeg.m

```diff
@@ -116,7 +116,7 @@
   end
    
   for ii=1:length(headmodel.bnd)
-    [junk,tname] = fileparts(tempname);
+    [junk,tname] = fileparts(['/tmp/mesh' num2str(ii)]);%; fileparts(tempname);
```

Comment:
this will not work on windows right?

Reply:
Good catch, I didn't mean to leave that hard-coded.

Reply:
Fixed in #742